libcontainer/rootfs_init_linux.go

```diff
@@ -265,7 +265,8 @@ func (l *linuxRootfsInit) Init() error {
 			if m.Flags&^(unix.MS_REC|unix.MS_REMOUNT|unix.MS_BIND) != 0 {
 				// only remount if unique mount options are set
 				if err := remount(m); err != nil {
-					return newSystemErrorWithCausef(err, "remount %s to %s", m.Source, m.Destination)
+					return newSystemErrorWithCausef(err, "remount of %s with flags %#x",
```

Comment:
Looking at the code-path, it's technically posible for a pure bind-mount operation to get us here, right? Shouldn't you leave the 'source' attribute being printed here too?

Reply:
The remount() function only remounts (i.e., always sets the MS_REMOUNT flag), so the source attribute gets ignored by the kernel. 

There are also two call sites for remount() (rootfs_linux.go:425 and rootfs_init_linux.go:267), and both call it after having performed an initial mount, so the remount never does anything except to remount.